backend/src/main/kotlin/org/darkSolace/muse/userModule/model/UserSettings.kt

```diff
@@ -0,0 +1,49 @@
+package org.darkSolace.muse.userModule.model
+
+import org.darkSolace.muse.layoutModule.model.FontFamily
+import org.darkSolace.muse.layoutModule.model.FontSize
+import org.darkSolace.muse.layoutModule.model.Layout
+import org.darkSolace.muse.storyModule.model.Rating
+import org.darkSolace.muse.storyModule.model.StoryTag
+import org.hibernate.Hibernate
+import javax.persistence.*
+
+@Entity
+data class UserSettings(
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    var id: Long? = null,
+    @OneToOne
+    var selectedLayout: Layout? = null,
+    var emailVisible: Boolean = false,
+    var birthdayVisible: Boolean = false,
+    var realNameVisible: Boolean = false,
+    var maxRating: Rating = Rating.PARENTAL_GUIDANCE_13,
```

Comment:
do we already need to set up a way to make the defaults configurable for the actual production environment?

Reply:
not needed yet